src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -131,11 +131,14 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
         win.renderQuerySummary(document.getElementById('main_smry_info_div'));
 
         // hide mutex tab
-        $(document).ready(()=>{
-            if (genes((window as any).serverVars.theQuery).length <= 1) {
-                $('a#mutex-result-tab').parent().hide();
-            }
-        });
+        if ((window as any).serverVars.theQuery.trim() != "") {
```

Comment:
i think it would be safer to put this inside the ready callback.  otherwise, there's potential exec order issues.

Reply:
actually, i'm not sure i understand this.  what is relationship here to genesets?

Reply:
Since now we're supporting queries without genes, `genes((window as any).serverVars.theQuery)` fails as `window.serverVars.theQuery` is empty.

Reply:
this logic is backwards - the mutex tab will never get hidden for a geneset query in this case

I think what you want is to put this on line 136: 
`if (!(window as any).serverVars.theQuery.trim().length || ...)`